nodebuilder/tests/da_test.go

```diff
@@ -37,8 +37,12 @@ func TestDaModule(t *testing.T) {
 	blobs := make([]*blob.Blob, 0, len(appBlobs0)+len(appBlobs1))
 	daBlobs := make([][]byte, 0, len(appBlobs0)+len(appBlobs1))
 
-	for _, b := range append(appBlobs0, appBlobs1...) {
-		blob, err := blob.NewBlob(b.ShareVersion, namespace, b.Data)
+	for _, squareBlob := range append(appBlobs0, appBlobs1...) {
+		blob, err := blob.NewBlob(
```

Comment:
Nit: looks like this is the same as `convert` from file above (`blob_test.go`). Should we use `convert` consistently?

Reply:
Good point! 9a59f26fb434e82585717b92ed937397dbdd9531

Reply:
I can't use `convert` because in integration tests, sometimes this file gets compiled without the blob_test.go file. See [this error](https://github.com/celestiaorg/celestia-node/actions/runs/10218577164/job/28275209848#step:4:7).

Reverted in 77d04779f81216015220ab5f0c3c835ee4a73f63